app/views/devise/unlocks/new.html.erb

```diff
@@ -1,12 +1,20 @@
-<h2>Resend unlock instructions</h2>
+<div class="redtop">
+  <div class="container">
+    <div class="row">
+      <div class="resend-unlock col-lg-4 col-lg-offset-4 col-sm-6 col-sm-offset-3">
```

Comment:
Can we change the columns to `col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2`?